### DISCUSSION TITLE
Netlify SSR [Vue warn]: Component <Anonymous> is missing template or render function.

### DISCUSSION THREAD

**Discussion by itsmnthn** [nuxt/nuxt#16917] [Questions]
I get this on a Nuxt3 project deployed on netlify with ssr on and I get this warning on server log
`Aug 28, 07:21:58 PM: f43802f2 WARN   [Vue warn]: Component <Anonymous> is missing template or render function.`

And on inspecting page source of the webpage there is no rendered HTML page renders all things client side similar to SSG but not even populating html metadata.

Source Code: https://github.com/itsmnthn/nuxt3-web3-onboard-mvp
Here is the deployed URL: https://verdant-meerkat-6551ef.netlify.app/

**Comment by danielroe** [comment]
Please update to the latest version of Nuxt + Nitropack. This issue has been fixed, I belive.

**Reply by itsmnthn** [reply]
I've updated same on [private project](https://dulcet-monstera-918628.netlify.app) but it seems not working check the page source. 1. did clear cache and deploy on netlify 2. delete lock file and reinstalled dependencies with specific version

here is `package.lock` files

```
{
  "devDependencies": {
    "@commitlint/cli": "^17.0.3",
    "@commitlint/config-conventional": "^17.0.3",
    "@intlify/unplugin-vue-i18n": "^0.5.0",
    "@nuxt/test-utils": "^3.0.0-rc.8",
    "@pinia/nuxt": "^0.4.1",
    "@types/lodash": "^4.14.184",
    "@types/qrcode": "^1.5.0",
    "@typescript-eslint/eslint-plugin": "^5.33.1",
    "@typescript-eslint/parser": "^5.33.1",
    "@unocss/nuxt": "^0.45.8",
    "@unocss/transformer-directives": "^0.45.8",
    "@vueuse/nuxt": "^9.1.0",
    "eslint": "^8.22.0",
    "eslint-config-prettier": "^8.5.0",
    "eslint-plugin-nuxt": "^3.2.0",
    "eslint-plugin-prettier": "^4.2.1",
    "eslint-plugin-vue": "^9.3.0",
    "husky": "^8.0.1",
    "nuxt": "^3.0.0-rc.8",
    "postcss-html": "^1.4.1",
    "prettier": "^2.7.1",
    "stylelint": "^14.11.0",
    "stylelint-config-prettier": "^9.0.3",
    "stylelint-config-standard": "^28.0.0",
    "typescript": "^4.7.4",
    "vitest": "^0.22.1",
    "vue-i18n": "^9.2.2"
  },
  "dependencies": {
    "@notionhq/client": "^2.1.1",
    "@unocss/reset": "^0.45.8",
    "@web3-onboard/core": "^2.8.0",
    "@web3-onboard/injected-wallets": "^2.1.0",
    "dayjs": "^1.11.5",
    "ethers": "^5.7.0",
    "html-to-image": "^1.10.4",
    "lottie-web": "^5.9.6",
    "pinia": "^2.0.20",
    "qrcode": "^1.5.1",
    "vue3-popper": "^1.5.0"
  },
  "engines": {
    "node": "^16.11.0 || ^17.0.0 || ^18.0.0"
  },
  "packageManager": "yarn@3.2.0",
  "workspaces": [
    "packages/*"
  ]
}

```

**Reply by danielroe** [reply]
In that case, would you open an issue with a minimal reproduction?

**Reply by itsmnthn** [reply]
Created issue with reproduction #7325

**Comment by itsmnthn** [comment]
Minimal reproduction of the issue: <https://stackblitz.com/edit/nuxt-starter-m56paj>
